Formula/imagemagick.rb

```diff
@@ -126,7 +122,8 @@ def install
     args << "--with-quantum-depth=#{quantum_depth}" if quantum_depth
 
     # versioned stuff in main tree is pointless for us
-    inreplace "configure", "${PACKAGE_NAME}-${PACKAGE_VERSION}", "${PACKAGE_NAME}"
+    inreplace "configure",
```

Comment:
>Not talking about the specific discussion thread there.

Yeah two space continuations in general are a bit weird, but I don't think rubocop understands alignment directly under the '"' unless we extend it, which hasn't happened yet.